tests/rpc/test_rpc_telegram.py

```diff
@@ -966,6 +966,19 @@ async def test_telegram_profit_handle(
     assert "*Expectancy (Ratio):*" in msg_mock.call_args_list[-1][0][0]
     assert "*Trading volume:* `126 USDT`" in msg_mock.call_args_list[-1][0][0]
 
+    msg_mock.reset_mock()
+    # Test /profit long
+    context.args = ["long"]
+    await telegram._profit(update=update, context=context)
+    assert msg_mock.call_count == 1
+    assert "*ROI (Long Trades):* All trades" in msg_mock.call_args_list[-1][0][0]
```

Comment:
Under the assumption that we have good reason to use `/profit long` instead of `/profit-long` ... 

You will **also** want to check that "/profit long n" does what it should ... so that numeric parsing works and it's filtered.

That obviously includes ensuring that `/profit long` returns a different value than `/profit long n` ... (or the long side, which ever has the better / easier test data) - to make sure that the numeric filter works on both the 2nd and 3rd place).